src/controller/model/endpoint.ts

```diff
@@ -739,7 +739,7 @@ class Endpoint extends Entity {
             });
         } catch (error) {
             const err = error as Error;
-            err.message = `${log} failed (${err.message})`;
+            err.message = `${messageBuilder()} failed (${err.message})`;
             logger.debug((err as Error).stack!, NS);
```

Comment:
Should cleanup these (even though not strictly related to this PR, looks like there are a few occurrences).
`(err as Error)` => `err` (already declared as such with `const err = error as Error;`)

Reply:
Fixed.